src/Solnet.Examples/StakeExample.cs

```diff
@@ -0,0 +1,322 @@
+                    StakeProgram.ProgramIdKey))
+                .AddInstruction(StakeProgram.Initialize(
+                    stakeAccount.PublicKey,
+                    authorized,
+                    lockup))
+                .Build(new List<Account> { fromAccount, stakeAccount});
+            Console.WriteLine($"Tx base64: {Convert.ToBase64String(tx)}");
+            RequestResult<ResponseValue<SimulationLogs>> txSim = rpcClient.SimulateTransaction(tx);
+
+            string logs = Examples.PrettyPrintTransactionSimulationLogs(txSim.Result.Value.Logs);
+            Console.WriteLine($"Transaction Simulation:\n\tError: {txSim.Result.Value.Error}\n\tLogs: \n" + logs);
+            RequestResult<string> firstSig = rpcClient.SendTransaction(tx);
+            Console.WriteLine($"First Tx Result: {firstSig.Result}");
+        }
+    }
+    public class MasterStakeBytesExample : IExample
+    {
+        private static readonly IRpcClient rpcClient = ClientFactory.GetClient(Cluster.TestNet);
+
+        private const string MnemonicWords =
+           "clerk shoe noise umbrella apple gold alien swap desert rubber truck okay twenty fiscal near talent drastic present leg put balcony leader access glimpse";
+
+        public void Run()
+        {
+            var wallet = new Wallet.Wallet(new Mnemonic(MnemonicWords));
+            rpcClient.RequestAirdrop(wallet.Account.PublicKey, 100_000_000);
+            RequestResult<ResponseValue<BlockHash>> blockHash = rpcClient.GetRecentBlockHash();
+            ulong minbalanceforexception = rpcClient.GetMinimumBalanceForRentExemption(StakeProgram.StakeAccountDataSize).Result;
+            
+            Account a6 = wallet.GetAccount(6);
+            Account a5 = wallet.GetAccount(5);
+            Account a4 = wallet.GetAccount(4);
+            Account a3 = wallet.GetAccount(3);
+            Account a2 = wallet.GetAccount(2);
+
+            Authorized authorized = new()
+            {
+                Staker = a5,
+                Withdrawer = a4
+            };
+            Lockup lockup = new()
+            {Custodian = a3.PublicKey,
+                Epoch = 0,
+                UnixTimestamp = 0
+            };
+
+            Console.WriteLine($"BlockHash >> {blockHash.Result.Value.Blockhash}");
+
+            byte[] tx = new TransactionBuilder()
+                .SetRecentBlockHash(blockHash.Result.Value.Blockhash)
+                .SetFeePayer(a6)
+                .AddInstruction(SystemProgram.CreateAccountWithSeed(
+                    a6.PublicKey,
+                    a5,
+                    a6,
+                    "dog1",
+                    3 * minbalanceforexception + 42,
+                    StakeProgram.StakeAccountDataSize,
+                    StakeProgram.ProgramIdKey))
+                .AddInstruction(SystemProgram.Transfer(
+                    a6,
+                    a5,
+                    5
+                    ))
+                //.AddInstruction(StakeProgram.Initialize(
```

Comment:
Remove commented code. These examples need to be clean, so that someone new coming to Solnet has some examples to follow and to help understand things. Commented code won't help.